theia-turn/turn-sprotty/src/views/ucm/startpoint.node.tsx

```diff
@@ -2,18 +2,23 @@
 
 import { svg } from 'snabbdom-jsx';
 import {IView, RenderingContext} from "sprotty/lib";
-import {MAPStartNode} from "../../models";
+import {TURNNode} from "../../models";
 import { VNode } from "snabbdom/vnode";
 
 export class StartPointNodeView implements IView {
 
     static readonly TYPE = 'turnnode:startpoint';
 
-    render(node: MAPStartNode, context: RenderingContext): VNode {
-        const radius = node.getDimension() / 2;
+    render(node: TURNNode, context: RenderingContext): VNode {
+        // const xOffset = Math.max(node.bounds.width, 0);
+        const yOffset = Math.max(node.bounds.height / 2, 0);
```

Comment:
This offset is creating a slight visual bug where the offsets for StartPoints and EndPoints are being calculated differently and thus they're offset from eachother when put side by side. Might be related to line 19.
<img width="614" alt="Capture" src="https://user-images.githubusercontent.com/25235305/69211389-ed3e3e00-0b2b-11ea-9eb9-f3a16f25d027.PNG">